.github/workflows/beta-deployment.yml

```diff
@@ -15,17 +15,18 @@ on:
   workflow_dispatch:
 
 jobs:
-  buildandtest:
-    name: Build and Test
-    uses: ./.github/workflows/build-and-test.yml
-    secrets: inherit
+  # buildandtest:
+  #   name: Build and Test
+  #   uses: ./.github/workflows/build-and-test.yml
+  #   secrets: inherit
```

Comment:
@PSchmiedmayer can I remove this Job completely, or what was the reason to uncomment that?